### DISCUSSION TITLE
How to center InSAR in my area

### DISCUSSION THREAD

**Discussion by cubarro** [insarlab/MintPy#985] [Q&A] [answered]
I have an area of interest (AOI) like this

![area](https://user-images.githubusercontent.com/26281001/226596348-b6f1fee2-f998-487d-8cec-d9a4212d5323.png)

But my InSAR analysis allways ends at north or south of my AOI

![finishes](https://user-images.githubusercontent.com/26281001/226597001-ebb97816-aafb-492c-bf17-a986605ffcfb.png)

How can I set to work in my AOI?

Regards

**Comment by jhkennedy** [comment, marked as answer]
@cubarro are you using On Demand processed InSAR data from ASF HyP3? And what reference scene are you using for your SBAS search?

**Reply by jhkennedy** [reply]
It would also help if you'd post the WKT of your area of interest.

**Reply by cubarro** [reply]
@jhkennedy you are right.  And I need to run the SBAS.  I am usign a box defined with these values

POLYGON((-122.2368 57.1955,-121.7371 57.1955,-121.7371 57.5097,-122.2368 57.5097,-122.2368 57.1955))

Reference scene is this
S1A_IW_SLC__1SDV_20170311T014819_20170311T014846_015638_019B88_6D90

**Reply by jhkennedy** [reply]
@cubarro you're area of interest (AOI) over-laps with w a single burst on the edge of your reference scene:
![image](https://user-images.githubusercontent.com/7882693/226938896-5c98aedc-b3d2-4911-836f-087ac974411a.png)

And, if in Vertex, you search for ascending, path 166, scenes that intersect your AOI, you'll notice that your AOI is generally right on the edges of scenes:
![image](https://user-images.githubusercontent.com/7882693/226939916-04d53bb6-dfda-4a46-ae09-029537f2404e.png)

Unfortunately, you'll get a lot of pairs using that reference scene that don't have overlaps due to the nodata padding around the valid data in the SLC GeoTIFF. And then when you subset to the valid data, you may be throwing away any data in your AOI due to most of the valid data being bursts interior to edges (you'll want to carefully check your list before subsetting). In ISCE2, you can include multiple reference and secondary scenes to ensure your AOI is always completely covered, but we don't currently offer that with HyP3. 

For HyP3 products, you can use the % overlap filter to ensure you do get overlapping bursts in your pairs:
![image](https://user-images.githubusercontent.com/7882693/226941787-998800ee-4d31-4c9b-9f15-d9c728aa5bd9.png)

You may also have better luck processing Path 64:
![image](https://user-images.githubusercontent.com/7882693/226942914-1ca4ebf2-d9b8-4aab-abe5-60aa7c682e81.png)

**Reply by cubarro** [reply]
That is an amazing solucion!

I will use it.

Thank you very much, @jhkennedy !!

**Reply by cubarro** [reply]
> @cubarro you're area of interest (AOI) over-laps with w a single burst on the edge of your reference scene: ![image](https://user-images.githubusercontent.com/7882693/226938896-5c98aedc-b3d2-4911-836f-087ac974411a.png)
> 
> And, if in Vertex, you search for ascending, path 166, scenes that intersect your AOI, you'll notice that your AOI is generally right on the edges of scenes: ![image](https://user-images.githubusercontent.com/7882693/226939916-04d53bb6-dfda-4a46-ae09-029537f2404e.png)
> 
> Unfortunately, you'll get a lot of pairs using that reference scene that don't have overlaps due to the nodata padding around the valid data in the SLC GeoTIFF. And then when you subset to the valid data, you may be throwing away any data in your AOI due to most of the valid data being bursts interior to edges (you'll want to carefully check your list before subsetting). In ISCE2, you can include multiple reference and secondary scenes to ensure your AOI is always completely covered, but we don't currently offer that with HyP3.
> 
> For HyP3 products, you can use the % overlap filter to ensure you do get overlapping bursts in your pairs: ![image](https://user-images.githubusercontent.com/7882693/226941787-998800ee-4d31-4c9b-9f15-d9c728aa5bd9.png)
> 
> You may also have better luck processing Path 64: ![image](https://user-images.githubusercontent.com/7882693/226942914-1ca4ebf2-d9b8-4aab-abe5-60aa7c682e81.png)

This answer already solved my problem.
Thank you very much, @jhkennedy 

Another question is how to find the convenent path...
I do not know if it should be addressed by another entry in Q&A.

Regards
